examples/studio-preview/.env.example

```
@@ -0,0 +1,2 @@
+OPENAI_API_KEY=
+MASTRA_PREVIEW_MODEL=__AI_SDK_OPENAI_MODEL_BASE__
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MASTRA_PREVIEW_MODEL key should go before the OPENAI_API_KEY key

(UnorderedKey)